### PR TITLE
[Whitesource] Bump coreos/etcd

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

`coreos/etcd` was to be bumped to 3.3.23+ and 3.4.10+.

**Related issue(s)**
[Bump](https://github.com/kyma-project/kyma/pull/9324)

### CODE REVIEW

**Comment by Wawrzyn321** [comment]
BRUH

File: components/console-backend-service/go.mod [hunk 1]
```diff
@@ -55,6 +55,7 @@ require (
 )
 
 replace (
+	github.com/coreos/etcd => github.com/coreos/etcd v3.3.24+incompatible
```
Comment (dariadomagala) [inline components/console-backend-service/go.mod:58]: Do we need this 'incompatible' part?

**Review by dariadomagala — APPROVED** [review]

**Pull request merged**